### PR TITLE
Initial version of bundle-size bot

### PR DESCRIPTION
**Pull request by danielrozenberg**

### CODE REVIEW

**Comment by danielrozenberg** [comment]
Missing from this PR:
* Lots of things
* Tests
* eslint rules, Travis checks, LGTM, etc...

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+**/node_modules/**
+.env
+package-lock.json
```
Comment (rsimha) [inline .gitignore]: Future cleanup note: If you're using `yarn`, this file should not be created. If you're using `npm`, this file should be checked in.

File: bundle-size/app.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+const {dbConnect} = require('./db');
+const path = require('path');
+
+const EMPTY_SHA = '0'.repeat(40);
+
+async function getBuildArtifactsFile(github, filename) {
+  if (filename == 'OWNERS') {
+    // Temporarily, until we add the OWNERS file to the build artifacts repo.
+    return 'choumx\ndanielrozenberg\njridgewell\nrsimha\n';
```
Comment (rsimha) [inline bundle-size/app.js]: Curious: Why not create an owners file right away and read from it?

File: bundle-size/app.js [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+    owner: 'ampproject',
+    repo: 'amphtml-build-artifacts',
+    path: path.join('bundle-size', filename),
+  }).then(result => {
+    return Buffer.from(result.data.content, 'base64').toString();
+  });
+}
+
+function formatBundleSizeDelta(delta) {
+  return 'Δ ' + (delta >= 0 ? '+' : '') + delta.toFixed(2) + 'KB';
```
Comment (rsimha) [inline bundle-size/app.js]: nit: Maybe use the `∇` symbol if the bundle size has reduced?

Comment (danielrozenberg) [reply bundle-size/app.js]: I prefer to stick with Δ, the mathematical notation for "difference" :)

File: bundle-size/app.js (hunk 2 above)
Comment (danielrozenberg) [reply bundle-size/app.js]: Will do and remove this line before submitting this PR

File: .gitignore (hunk 1 above)
Comment (danielrozenberg) [reply .gitignore]: I could use help figuring out the "infra" for this, let's talk offline

**Pull request merged**